contracts/marketplace/src/testing/tests/asks.rs

```diff
@@ -272,6 +272,23 @@ fn max_set_ask_amount() {
         &listing_funds(LISTING_FEE).unwrap(),
     );
     assert!(res.is_ok());
+
+    // Ask cannot be updated above the limit
+    let update_ask_price = ExecuteMsg::UpdateAskPrice {
+        collection: collection.to_string(),
+        token_id,
+        price: coin(100_000_000_000_001u128, NATIVE_DENOM),
```

Comment:
could import `MAX_FIXED_PRICE_ASK_AMOUNT` so it's `MAX_FIXED_PRICE_ASK_AMOUNT + 1` in the test